### PR TITLE
Add hybrid search for v1.6.0

### PR DESCRIPTION
**Pull request by brunoocasali**
Adds hybrid search to PHP SDK. 

There is a commented test that should be reverted as soon as the related issue is fixed on the engine.

### CODE REVIEW

File: src/Endpoints/Delegates/HandlesSettings.php [hunk 1]
```diff
@@ -361,4 +361,21 @@ public function resetProximityPrecision(): array
...
     }
+
+    // Settings - Experimental: Embedders (hybrid search)
+
+    public function getEmbedders(): array
+    {
+        return $this->http->get(self::PATH.'/'.$this->uid.'/settings/embedders');
+    }
+
+    public function updateEmbedders(array $embedders): array
```
Comment (norkunas) [inline src/Endpoints/Delegates/HandlesSettings.php:372]: `$embedders` structure should be defined

Comment (curquiza) [reply src/Endpoints/Delegates/HandlesSettings.php:372]: @norkunas we don't have the time to do it for this PR, can we open a PR for the community to update it?

Comment (norkunas) [reply src/Endpoints/Delegates/HandlesSettings.php:372]: yup, but I don't find anywhere docs about this :)

Comment (norkunas) [reply src/Endpoints/Delegates/HandlesSettings.php:372]: @curquiza from https://www.meilisearch.com/docs/learn/experimental/vector_search and https://github.com/meilisearch/meilisearch/releases/tag/v1.6.0 I understand that structure should be like: `array<non-empty-string, array{source: non-empty-string, apiKey?: non-empty-string, model?: non-empty-string, documentTemplate?: non-empty-string, dimensions: positive-int}>` can you confirm?

**Pull request merged**